src/main/java/pl/javastart/exercise/streams/FilteringMachine.java

```diff
@@ -1,23 +1,56 @@
 package pl.javastart.exercise.streams;
 
+import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class FilteringMachine {
 
     public List<Integer> filterOutNotEvenNumbers(List<Integer> numberList) {
-        return null;
+        List<Integer> notEvenNumbers = numberList.stream()
```

Comment:
Jak wynik streama od razu podajesz do sekcji return to nie musisz przypisywać wyniku do zmiennej. Możesz od razu zacząć pisać to w return. To się oczywiście tyczy wszystkich metod niżej.